client/verta/verta/endpoint/build/_build_scan.py

```diff
@@ -101,9 +101,11 @@ def _get(cls, conn: _utils.Connection, build_id: int):
         return cls(response.json())
 
     @classmethod
-    def _create(cls, conn: _utils.Connection, build_id: int, external: bool) -> "BuildScan":
+    def _create(
+        cls, conn: _utils.Connection, build_id: int, external: bool, workspace: str
```

Comment:
Here I'd actually recommend having `workspace` come right after `conn` to match all of our other creation methods.

Reply:
Also done.